back/internal/postsvc/internal/postdb/qrypost.go

```diff
@@ -61,16 +67,21 @@ func (s *PostDB) upsertPost(ctx cx, sid string, x *pb.AddPostReq, y *pb.PostResp
 		return err
 	}
 
-	_, err = stmt.Exec(id, x.typeId, x.title, x.body)
+	_, err = stmt.Exec(id, x.TypeId, x.Title, x.Body)
+
+	if err != nil {
```

Comment:
nit: This could be a single line assignment/conditional (please drop the empty line if not).

Reply:
Do you mean to change it to the following?
```
if _, err = stmt.Exec(id, x.TypeId, x.Title, x.Body); err != nil {
    return err
}
```